src/routers/OneTimePassword.ts

```diff
@@ -1,9 +1,10 @@
 import express from "express";
 import OTPService from "@/services/OneTimePassword";
+import TOTPConfig from "@/types/TOTPConfig";
 
 const router: express.Router = express.Router();
 
-router.use("/:patientId", async (req, res) => {
+router.use("/setup/:patientId", async (req, res) => {
```

Comment:
Please specify explicitly which methods this endpoint accepts. Please correct that for ALL endpoints. even though they are not part of this PR. Seems like I've missed that in other reviews. i.E.: 
```suggestion
router.post("/setup/:patientId", async (req, res) => {
```